### PR TITLE
Create config/buildnum with correct content

### PR DESCRIPTION
**Pull request by walterdolce**

### CODE REVIEW

File: openpassword/agile_keychain/data_source.py [hunk 1]
```diff
@@ -13,11 +13,18 @@ class DataSource(abstract.DataSource):
...
         self._default_folder = os.path.join(self._base_path, "data", "default")
+        self._config_folder = os.path.join(self._base_path, "config")
         self._key_manager = key_manager(self._base_path)
         self._keys = []
 
     def initialise(self, password):
         os.makedirs(self._default_folder)
+        os.makedirs(self._config_folder)
+
+        buildnum_file = os.path.join(self._config_folder, 'buildnum')
```
Comment (mlopes) [inline openpassword/agile_keychain/data_source.py]: Can I suggest this to be extracted to a `create_buildnum_file` method or something of the sort?

Comment (mlopes) [inline openpassword/agile_keychain/data_source.py]: I think the file name and the '32009' are not very intuitive, would be a good idea to have a static property for this. :)

**Comment by mlopes** [comment]
Closes #92

**Comment by nnevala** [comment]
Replaced by #95

**Pull request closed without merging**